livekit-plugins/livekit-plugins-deepgram/livekit/plugins/deepgram/stt.py

```diff
@@ -208,19 +202,19 @@ async def _run(self, max_retry: int) -> None:
             while not self._closed:
                 try:
                     live_config = {
-                        "model": self._config.model,
-                        "punctuate": self._config.punctuate,
-                        "smart_format": self._config.smart_format,
-                        "interim_results": self._config.interim_results,
+                        "model": self._opts.model,
+                        "punctuate": self._opts.punctuate,
+                        "smart_format": self._opts.smart_format,
+                        "interim_results": self._opts.interim_results,
                         "encoding": "linear16",
                         "sample_rate": self._sample_rate,
                         "vad_events": True,
                         "channels": self._num_channels,
-                        "endpointing": self._config.endpointing,
+                        "endpointing": self._opts.endpointing,
```

Comment:
IMO we get rid of endpointing because it's not coupled to transcription events and order is not guaranteed. It's more like VAD and it causes more issues than it helps